### DISCUSSION TITLE
Storing sensitive information using next-auth

### DISCUSSION THREAD

**Discussion by muftisamiullah** [nextauthjs/next-auth#2107] [Help]
## Question 💬

I just wanted to know how can we store senstive information using next-auth, i am saving accessToken in session and thats visible in network i just want to store somewhere can you tell me how i can do that using next-auth

```

```

**Comment by balazsorban44** [comment]
We provide two ways of storing session information either in a JWT directly in a session cookie or in a database. both have advantages and disadvantages. 

https://next-auth.js.org/faq#json-web-tokens have a look at this, and try browsing the documentation, see if it helps. :slightly_smiling_face:
